### DISCUSSION TITLE
Use Quasar Flex Grid in another project

### DISCUSSION THREAD

**Discussion by stevebor1** [quasarframework/quasar#14924] [General - Components / Directives / etc]
As a once avid Quasar user and now, sigh, Angular user, I am lacking a nice css library that simplifies Flex Grid (Google just deprecated angular flex-layout). People are floating around Tailwind CSS, but I keep thinking of how nice Quasar Flex Grid is. 

Is there any way I can use this part of the library directly in my project?

**Comment by MilosPaunovic** [comment]
You could try importing this in your HTML file:

`<link href="https://cdn.jsdelivr.net/npm/quasar@2/dist/quasar.prod.css" rel="stylesheet" type="text/css">`

But it might me an overkill, beacuse you could create basic CSS grid yourself pretty easy.
